website/notifications/tasks.py

```diff
@@ -20,6 +21,7 @@ def send_users_email(send_type):
     :param send_type
     :return:
     """
+    # Send emails to users, excluding reviews moderators
```

Comment:
The phrasing of this comment makes it sound like it will send emails to all users that are not reviews moderators. Might be clearer to state that it does not send moderation-triggered emails or something.

Reply:
That was actually an incorrect comment. Now it does send emails to moderators.